lib/graphql/schema.flow.js

```diff
@@ -1257,6 +1261,7 @@ export type MutationUpsertProductsArgs = {|
 
 export type MutationCategorizeTransactionForDeclarationArgs = {|
   id: $ElementType<Scalars, 'ID'>,
+  categoryCode?: ?$ElementType<Scalars, 'String'>,
   elsterCode?: ?$ElementType<Scalars, 'String'>,
```

Comment:
The file is automatically generated. Because the backend supports backward compatibility, therefore we still have `elsterCode` in request/response.